### PR TITLE
update context-menu-extensions to SDK 157444

### PR DESCRIPTION
**Pull request by DarthWeirdo**

### CODE REVIEW

File: kotlin/context-menu-extension/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (ark-1) [inline kotlin/context-menu-extension/.idea/.gitignore]: Are files from .idea added intentionally?

File: kotlin/context-menu-extension/src/main/kotlin/com/example/processing/MenuActionPayload.kt [hunk 2]
```diff
@@ -88,6 +107,6 @@ private suspend fun doCreateSubItems(client: SpaceClient, payload: MenuActionPay
 //            context.projectIdentifier,
 //            title = "${issue.title}: $it",
 //            status = status,
-//            parents = listOf(context.issueIdentifier)
+//            parents = listOf(context.issueIdentifier))
```
Comment (ark-1) [inline kotlin/context-menu-extension/src/main/kotlin/com/example/processing/MenuActionPayload.kt:110]: A typo?

File: kotlin/context-menu-extension/.idea/.gitignore (hunk 1 above)
Comment (denzakharov) [reply kotlin/context-menu-extension/.idea/.gitignore:1]: I had the same question :) @DarthWeirdo wrote in DM that his IDEA was set up to use JDK 8 for Gradle, while our SDK is compiled to be run on JDK 11 or newer. And he had an error on runtime. I don't think it's a good reason to include any .idea file here.

File: kotlin/context-menu-extension/src/main/kotlin/com/example/processing/MenuActionPayload.kt (hunk 2 above)
Comment (DarthWeirdo) [reply kotlin/context-menu-extension/src/main/kotlin/com/example/processing/MenuActionPayload.kt:110]: No, it's on purpose. It belongs to `client.projects.planning.issues.createIssue(` 
I guess there was a typo in the original commented code

File: kotlin/context-menu-extension/.idea/.gitignore (hunk 1 above)
Comment (DarthWeirdo) [reply kotlin/context-menu-extension/.idea/.gitignore]: removed it

**Review by denzakharov — APPROVED** [review]

**Pull request merged**